_commands/basics/chmod.md

```diff
@@ -0,0 +1,65 @@
+---
+---
+
+chmod
+-------
+
+Changes the permissions of a file or directory.
+Stands for **ch**ange file **m**od **b**its
+
+~~~ bash
+$ ls -go --time-style=+
+total 40
+-rw-r--r-- 1 37465  index.html
+$ chmod 666 index.html
+$ ls -go --time-style=+
+total 40
+-rw-rw-rw- 1 74981  index.html
+~~~
+
+<!--more-->
+
+### Details
+The first argument to `chmod` determines the bitset of permissions you want to change the file given in the second argument.
+
+The bitset should correspond to rwx (read write exectue) permissions. e.g. 6 becomes 110 which is read-able write-able and not executable
```

Comment:
1. I've never heard them called bitset – usually [modes](https://en.wikipedia.org/wiki/Modes_(Unix)), but if you can find documentation otherwise that's totally fine.
2. I think it'd be more clear to reverse the example here: "binary 110 becomes base-10 6..."